TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/DriveConstants.java

```diff
@@ -34,7 +34,12 @@
      * Set the value of MOTOR_VELO_PID to `new PIDCoefficients(kP, kI, kD);`
      */
     public static final boolean RUN_USING_ENCODER = true;
-    public static final PIDCoefficients MOTOR_VELO_PID = null;
+
+    public static double kP = 0;
+    public static double kI = 0;
+    public static double kD = 0;
+
+    public static final PIDCoefficients MOTOR_VELO_PID = new PIDCoefficients(kP, kI, kD);
```

Comment:
I'd prefer that these values aren't duplicated in two different places. I usually prefer the POJO in this situation.

Reply:
So the rationale for this was to make the gains persist between Opmode lifecycles. This is most helpful in the event that your bot is on course to hit an object. It is inevitable that your bot goes off path because it tends to drift to one side over the course of the tuning process (whether it be from the untuned motors causing the drift or uneven weight distribution or whatever). But you do find yourself hitting the stop button quite often and in the event that you are unable to save your gains, it is frustrating when you lose them.  

So having the values persist between opmodes requires static fields, as far as I'm aware? Is there any way of retaining the functionality while following your best practices?

Reply:
Sorry I should have made my response more clear. Make `MOTOR_VELO_PID` non-`final` and use that to persist the values.

Reply:
Oops. I should have thought of just removing final. My apologies.